src/DGtal/shapes/Mesh.ih

```diff
@@ -256,6 +257,20 @@ DGtal::Mesh<TPoint>::getFace(unsigned int i)
 
 template<typename TPoint>
 inline
+TPoint 
+DGtal::Mesh<TPoint>::getFaceBarycenter(unsigned int i) const
+{
+  TPoint c; 
+  MeshFace aFace = getFace(i);
+  for ( auto &j: aFace){
+    c += getVertex(j);
+  }
+  return c/static_cast<typename TPoint::Component>(aFace.size());
```

Comment:
well.. yes.. but If the mesh has integer coordinates vertices, the barycenter is a bit crappy since it may not lie on the surface.